asab/zookeeper/container.py

```diff
@@ -21,19 +21,25 @@ class ZooKeeperContainer(ConfigObject):
 		# Server list to which ZooKeeper Client tries connecting.
 		# Specify a comma (,) separated server list.
 		# A server is defined as address:port format.
-		"servers": "zookeeper-1:2181,zookeeper-2:2181,zookeeper-3:2181",
-
-		"path": "/asab",
+		# Example:
+		# "servers": "zookeeper-1:2181,zookeeper-2:2181,zookeeper-3:2181"
+		#
+		# WARNING: If the value is not provided (empty string), Zookeeper container refuses to start
+		"servers": "",
```

Comment:
@ateska Now the confusion is completely removed. Nice.

Reply:
@ateska I tested it without the servers works as intended.

 **_CRITICAL asab.zookeeper Cannot connect to Zookeeper, the configuration value of 'servers' not provided._**
 
 On the other side if I do not provide the path he will use the root directory of zookeeper. 
 Would it be nice to throw a warning like 
 
 **Path is not passed, using the root.**
 
 Something like that?

Reply:
I don't want to warn people of something that could be ok.